### DISCUSSION TITLE
Secure Onboarding

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1470] [Office hours] [answered]
For design system office hours wanted to review this design one last time to see if theres anything that stands out as a red flag. 

One specific area I wanted to focus on was on step 3 choose a email sending option. Specifically the cards to make sure this is an appropriate use of the components. 

Figma link: https://www.figma.com/file/WTlMlYU1gEcCceHmdviNsQ/Secure-Onboarding?node-id=591%3A6

![Desktop - 61](https://user-images.githubusercontent.com/81593493/118844713-6a18b800-b888-11eb-81cf-1a4164ec9887.jpg)

**Comment by wilfredomerced** [comment]
adding figma link https://www.figma.com/file/WTlMlYU1gEcCceHmdviNsQ/Secure-Onboarding?node-id=591%3A6

**Comment by kate-porter** [comment]
Wow @wilfredomerced, these designs look so great! 😍 We'll all be sure to review before office hours later today. Looking forward to it!

**Comment by kate-porter** [comment, marked as answer]
Hey @wilfredomerced - just wanted to share the office hours recording with you in case you'd like to revisit any of our conversations. [Here's the video](https://www.loom.com/share/297fac81bffa4082ad8be0a26101c4a6) (it includes timestamps for each topic), and the password is posted in the #help-design-system channel. Thanks again for coming to office hours!
